src/core/View.js

```diff
@@ -324,9 +324,9 @@ export default class View extends Component {
       visibility: false,
       dataBounds: [-1, 1, -1, 1, -1, 1],
     });
-    this.cubeAxes
-      .getActors()
-      .forEach(({ setVisibility }) => setVisibility(false));
+    Array.from(this.cubeAxes.getActors()).forEach(({ setVisibility }) =>
+      setVisibility(false)
+    );
     this.cubeAxes.setCamera(this.camera);
```

Comment:
Why do you need `Array.from()`

Reply:
I was hitting an error where `getActors()` was returning `null` and `forEach` works on arrays. This change ensures that it is always an array.